code/modules/shuttle/super_cruise/orbital_poi_generator/objective_types/recover_blackbox.dm

```diff
@@ -4,8 +4,8 @@
 	//The blackbox required to recover.
 	var/obj/item/blackbox/objective/linked_blackbox
 	//Relatively easy mission.
-	min_payout = 50000	//10k credits for sci/sec/eng, 5k for ser / civ
-	max_payout = 100000	//20k credits for sci/sec/eng, 10k for serv / civ
+	min_payout = 5000	//10k credits for sci/sec/eng, 5k for ser / civ
+	max_payout = 10000	//20k credits for sci/sec/eng, 10k for serv / civ
```

Comment:
```suggestion
	min_payout = 5000	//1k credits for sci/sec/eng, 500 for ser / civ
	max_payout = 10000	//2k credits for sci/sec/eng, 1k for serv / civ
```

Presumably the comments should also have the numbers reduced tenfold?

Reply:
Yes, I'll do it when I am able to today